### PR TITLE
Support for custom size vms - based on 0.3.5

### PR DESCRIPTION
**Pull request by gmileka**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -24,3 +24,5 @@ require (
 )
 
 replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.4.0+incompatible
+
+replace github.com/microsoft/moc-sdk-for-go v0.10.1-alpha.1 => ../moc-sdk-for-go
```
Comment (gmileka) [inline go.mod:28]: This is necessary since HardwordProfile.CustomSize did not exist in v0.10.1-alpha.1.

File: go.mod [hunk 2]
```diff
@@ -7,7 +7,7 @@ require (
 	github.com/Azure/go-autorest/autorest/to v0.3.0
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/go-logr/logr v0.1.0
-	github.com/microsoft/moc v0.10.1-alpha.4
+	github.com/microsoft/moc v0.10.9
```
Comment (gmileka) [inline go.mod:10]: This is auto-changed by the build. It is a side effect of redirecting to latest moc-sdk-for-go.

File: go.mod (hunk 1 above)
Comment (bfjelds) [reply go.mod:28]: do we have a branch for moc-sdk-for-go?  if so, we can update go.mod to point at the CustomSize commit.

Comment (gmileka) [reply go.mod:28]: I have to point go.mod to the commit of moc-sdk-for-go that has the CustomSize. I'll do that change in the next round.

**Review by bfjelds — APPROVED** [review]

**Pull request merged**